generators/chipyard/src/main/scala/config/RocketConfigs.scala

```diff
@@ -21,6 +21,11 @@ class TinyRocketConfig extends Config(
   new freechips.rocketchip.subsystem.With1TinyCore ++             // single tiny rocket-core
   new chipyard.config.AbstractConfig)
 
+class FFTRocketConfig extends Config(
+  new fftgenerator.WithFFTGenerator(baseAddr=0x2000, numPoints=8, width=16, decPt=8) ++
```

Comment:
Can you add a comment to this line saying what it does (like what the other configs do)